lib/preview.js

```diff
@@ -162,21 +164,19 @@ const startLocalHttpService = async (sessionInfo) => {
   //expressServer.use(bodyParser.json());
   expressServer.use('/', await createPreviewServerRoutes(sessionInfo));
 
-  const httpServer = http.createServer(expressServer);
-  httpServer.listen(3000);
-  console.log('Now listening on port 3000');
-  return httpServer;
+  return expressServer;
```

Comment:
If we're using HTTPS we instead want to pass this handler over to that logic instead of creating the server & listening on the port, hence moving it from here.